src/ci/docker/x86_64-gnu-nopt/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   cmake \
   sudo \
   gdb \
+  p7zip-full \
```

Comment:
A bunch of containers don't actually produce tarballs, only the `dist-*` ones do I believe. Could this package be left out of all the containers (like this one) not producing tarballs?